src/lib/db.js

```diff
@@ -969,7 +1029,9 @@ export default class db {
       });
     });
   }
-
+  
+  // Refactoring suggestion: rename to `addAuthor`
+  // to follow the style of the rest of the code
```

Comment:
@emilgoldsmith I will open an issue on refactoring falcor and db instead. The files are extremely long and could use refactoring as to that as well.